DataMigration/src/test/java/org/sagebionetworks/dian/datamigration/PasswordGeneratorTests.java

```diff
@@ -0,0 +1,42 @@
+package org.sagebionetworks.dian.datamigration;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class PasswordGeneratorTests {
+    @Test
+    public void test_createBridgePassword() throws IOException {
+        // Test 10000 bridge passwords for validity
+        for (int i = 0; i < 10000; i++) {
+            String password = PasswordGenerator.INSTANCE.nextPassword();
+            assertNotNull(password);
+            assertEquals(9, password.length());
+            assertTrue(isValidBridgePassword(password));
+        }
+    }
+
+    public boolean isValidBridgePassword(String password) {
+        boolean containsUppercase = false;
+        boolean containsLowercase = false;
+        boolean containsNumeric = false;
+        boolean containsSpecial = false;
+
+        for (int i = 0; i < password.length(); i++) {
+            String character = Character.toString(password.charAt(i));
+            containsUppercase |= PasswordGenerator.UPPERCASE.contains(character);
```

Comment:
On these four lines I believe the correct operator for boolean disjunction should be `||=`, not `|=`, which is the bitwise operator.

Reply:
Oh right! Apparently ||= doesn't exist, at least in Java 8.  I switched to this format:

`containsUppercase = containsUppercase || PasswordGenerator.UPPERCASE.contains(character);`

Reply:
I was wrong.  There is no `||=`. So your change is the best sol'n.